packages/create-block/lib/prompts.js

```diff
@@ -59,8 +59,7 @@ const dashicon = {
 		return true;
 	},
 	filter( input ) {
-		return input &&
-			input.replace( /dashicon(s)?-/, '' );
+		return input && input.replace( /dashicon(s)?-/, '' );
 	},
 };
 
```

Comment:
I missed it the first time around, but just to be aware: There are proposed revisions to the set of block categories being discussed/proposed at #19279.

I have no good solutions to offer, but it would be nice if we could avoid hard-coding this. I guess we can just come back and update it when the time comes.